contrib/kind.sh

```diff
@@ -358,7 +358,7 @@ pushd ../dist/yaml
 run_kubectl apply -f k8s.ovn.org_egressfirewalls.yaml
 run_kubectl apply -f k8s.ovn.org_egressips.yaml
 run_kubectl apply -f ovn-setup.yaml
-MASTER_NODES=$(kind get nodes --name ${KIND_CLUSTER_NAME} | head -n ${KIND_NUM_MASTER})
+MASTER_NODES=$(kind get nodes --name ${KIND_CLUSTER_NAME} | sort | head -n ${KIND_NUM_MASTER})
```

Comment:
This works, but seems a bit hidden on what is going on. Should it instead ` | grep "control-plane" ` so if someone changes the node names (not likely I'm sure) they can see it is being used here?

Reply:
kind node names can not be change.
The main purpose of this changes was to get rid of the 3 nodes etcd cluster, that is not really necessary for ovn, because ovn HA is orthogonal to kubernetes HA.
So, right now, for an OVN HA cluster, we have ovn-control-plane, ovn-worker and ovn-worker2, with the only difference that the workers now have ovn master components

Reply:
ack

Reply:
/lgtm